src/test/resources/callstacks/TestRegressionGlobalVar_globalVar.txt

```diff
@@ -0,0 +1,8 @@
+   globalVar.run()
+      globalVar.node(groovy.lang.Closure)
+         globalVar.stage(One, groovy.lang.Closure)
+            globalVar.echo(Stage One)
+            globalVar.doWithProperties({PROP_1=VAL_1})
```

Comment:
If you don't apply the code fix, this call stack here in the first stage will show:

`globalVar.doWithProperties({PROP_2=VAL_2, PROP_1=VAL_1})`

At this point in the execution, that would clearly not be correct as PROP_2 is added in the second stage.